src/styles/contacts.styl

```diff
@@ -120,4 +120,13 @@ $reset-list
 
 .contacts-empty-soon
     display flex
-    justify-content center
\ No newline at end of file
+    justify-content center
+
+
+.fil-path-title
```

Comment:
None of the [cozy-ui text components](https://docs.cozy.io/cozy-ui/react/#text) can do the job? [Or the utility classes](https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html)?

If not, are we missing a size in our component list?

Reply:
I used the title component at the end. I'll check if i can remove some css properties. I just c/p from drive to be franc...